qoqo_qiskit/pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[project]
+name = "qoqo_qiskit"
+version = "0.1.0"
+authors = [{name="HQS Quantum Simulation GmbH", email="info@quantumsimulations.de"}]
+readme = "README.md"
+requires-python = ">=3.7"
+dependencies = [
+    "qoqo>=1.2",
+    "qoqo_qasm>=0.5",
+    "qiskit",
+    "numpy"
+]
+[project.optional-dependencies]
+tests = [
+  'pytest',
+]
+dev = [
+    'flake8',
+    'flake8-bugbear',
+    'flake8-pyproject',
+    'flake8-pydocstyle',
+    'darglint',
+    'bandit',
+    'mypy',
+    'black'
+]
+
```

Comment:
We should add an additional docs = [ ...] optional feature to get rid of the requirements.txt in the docs.